GitCommands/Git/GitModule.cs

```diff
@@ -757,7 +765,12 @@ public IReadOnlyList<string> GetSortedRefs()
                 "refs/"
             };
 
-            var tree = _gitExecutable.GetOutput(args);
+            string tree = _gitExecutable.GetOutput(args);
+            int warningPos = tree.IndexOf("warning:");
```

Comment:
is this ever localized? Anyone that has checked Git source for this?

Reply:
We use the same literal in at least one other place in the code, so I presume it is safe to use.
However it won't hurt to check

Reply:
"ignoring broken ref %s" is localized but not "warning: ":
```c
static void warn_builtin(const char *warn, va_list params)
{
	vreportf("warning: ", warn, params);
}
```